apps/docs/src/components/Header/Header.astro

```diff
@@ -41,20 +41,13 @@ const lang = getLanguageFromURL(currentPage);
         </h1>
       </a>
     </div>
-    <div class="grow"></div>
     {
       KNOWN_LANGUAGE_CODES.length > 1 && (
         <LanguageSelect lang={lang} client:idle />
       )
     }
-    {
-      /*     
-    <div
-      class="flex relative z-10 grow pr-[0.7rem] max-w-[200px] md:max-w-[400px]"
-    >
-      <Search client:idle />
-    </div> 
-    */
-    }
+    <div class="my-[2rem]">
```

Comment:
I don't think that is necessary, also please don't use arbitrary values and stick to the grid

Reply:
Right, done